src/lib/FlightTasks/tasks/FlightTaskAutoLine.cpp

```diff
@@ -45,112 +45,12 @@ using namespace matrix;
 
 bool FlightTaskAutoLine::activate()
 {
-	bool ret = FlightTaskAuto::activate();
-	_reset();
-	return ret;
+	return FlightTaskAutoMapper::activate();
```

Comment:
if that is the only thing that FlightTaskAutoLine is doing, then it is not necessary to override FlightTaskAutoMapper::activate()